Dockerfile

```diff
@@ -3,7 +3,7 @@ FROM python:3.8-slim
 ENV PIP_NO_CACHE_DIR=1 \
     JAVA_HOME=/usr/lib/jvm/java-11-openjdk-amd64/ \
     PYTHONPATH=/app \
-    SIGNAL_CLI_VERSION=0.6.10
+    SIGNAL_CLI_VERSION=0.7.0
```

Comment:
I dunno this seems like a lot to change in one go, are you sure we can't break this up into smaller, more digestible changes? lol.

Reply:
Yea sorry, it was just easier to have this all as 1 PR. I know it's a lot to review, thanks for taking the time to do it :wink: